landscape.yml

```diff
@@ -1975,6 +1975,23 @@ landscape:
               artwork_url: https://github.com/cncf/artwork/blob/master/examples/incubating.md#cilium-logos
               stack_overflow_url: https://stackoverflow.com/questions/tagged/cilium
               slack_url: https://cilium.herokuapp.com/
+              summary_personas: Platform engineers, Network teams, SRE, DevOps Engineers, Security teams
```

Comment:
Is this change intended?

Reply:
That's not mine - I merged it with the rest of the changes today when rebasing :)

Reply:
Ok I think you'll want to make sure the diff only contains the changes you made. Probably need to do a git pull --rebase. Perhaps the easiest way for you is to create a brand new fork/branch and send a new PR.